src/Sylius/Bundle/RbacBundle/spec/Form/EventSubscriber/AddParentFormSubscriberSpec.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\RbacBundle\Form\EventSubscriber;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\RbacBundle\Form\EventSubscriber\AddParentFormSubscriber;
+use Sylius\Component\Rbac\Model\RoleInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormInterface;
+use Sylius\Bundle\RbacBundle\Form\Exception\UnexpectedTypeException;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class AddParentFormSubscriberSpec extends ObjectBehavior
+{
+    function let()
+    {
+       $this->beConstructedWith('role');
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(AddParentFormSubscriber::class);
```

Comment:
You should use FQCN in this one instead of `::class` as stated in https://github.com/Sylius/Sylius/pull/3736#issuecomment-165732890